workflows/sars-cov-2-variant-calling/sars-cov-2-pe-illumina-artic-variant-calling/pe-artic-variation.ga

```diff
@@ -312,15 +294,20 @@
             "type": "tool",
             "uuid": "849d9cac-14b8-4e45-823d-5747709e8b60",
             "workflow_outputs": [
-                {
-                    "label": "fastp_reads_output",
-                    "output_name": "output_paired_coll",
-                    "uuid": "38598192-0867-4b52-ae7e-e7017259a1d6"
-                },
                 {
                     "label": "fastp_html_report",
                     "output_name": "report_html",
-                    "uuid": "085c3083-2258-4673-90b6-22fe095cd4ad"
+                    "uuid": "341566b1-181a-4bb0-9d0f-37db45003b22"
+                },
+                {
+                    "label": "fastp_json_report",
+                    "output_name": "report_json",
+                    "uuid": "04955ac5-df51-41ff-bd19-a0a99c7ef761"
```

Comment:
@mvdbeek I didn't even know this is possible. This adds the json report of fastp to the workflow outputs, but above the post job action of hiding that dataset is still defined. So these two things *can* be decoupled safely (just not through the GUI)?

Reply:
Yep, I thought that's possible

Reply:
It seems to work. The WF editor shows it as a workflow output, but when running the WF the dataset collection gets hidden. Nice!

Reply:
I often edit the file directly to produce nicer diffs here. It's maybe not even more dangerous than using the WF editor, which tends to drop different things in different Galaxy versions ;)